rpa-manager/modules/ROOT/pages/processautomation-manage-botworkflowlibrary.adoc

```diff
@@ -0,0 +1,109 @@
+= Managing the Bot Workflow Library
+
+The *Bot Workflow Library* view shows all bot workflows from the RPA Builder Activity Library.
+
+You can view and manage the workflows:
+
+* Create, edit, and delete tags for the library.
+* View an outline of a workflow.
+* Edit the description and the tags of a workflow.
+
+== Before You Begin
+
+* Ask an organization administrator in Access Management to assign you the required permissions:
++
+[cols="1,1,1"]
+|===
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
+
+|Open the *Process Automation* module
+|RPA Administrator
+|Process Automation Open
+
+|Open the Bot Workflow Library
+|RPA Automations Contributor, RPA Automations Manager
+|Activity Library Open
+
+|Create tags for the library. Add, edit, and remove tags and descriptions from bot workflows
```

Comment:
```suggestion
|Create tags for the library. Add, edit, and remove tags and descriptions from bot workflows.
```

Reply:
I think that this would be better as bullet list items, which would make this more skimmable.